app/views/grant_submissions/submissions/_section_fields.html.erb

```diff
@@ -31,7 +31,7 @@
                                      options: { rows: 1, cols: 30, data: { question_id: ff.object.question.id } }) %>
                 <% when :long_text %>
                     <%= ff.text_area(attribute: ffn,
-                                     options: { rows: 5, cols: 31, required: ff.object.question.is_mandatory, data: { question_id: ff.object.question.id } }) %>
```

Comment:
ah, this is the front-end validation I missed. So I guess for the other ticket to work you would have not use HTML built-in form validation or use the JavaScript validation API to customize the behavior by state.